### DISCUSSION TITLE
Bug: renaming a branch closes pull requests to other repositories

### DISCUSSION THREAD

**Discussion by hagai-helman** [community/community#5566] [Pull Requests] [answered]
(Note: this is a different bug than #5499)

I recently forked [deehzee/unionfind](https://github.com/deehzee/unionfind), and opened a [pull request](https://github.com/deehzee/unionfind/pull/5) from my fork's main branch `master` to the original repository main branch `master`. Later, I changed my branch name to `main`. This closed my pull request, and I was not able to reopen it.

**Comment by MathiasReker** [comment, marked as answer]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by hagai-helman** [reply]
> Hello. You can report the bug here: https://support.github.com/contact/bug-report

Thanks, Mathias!

Since long time has passed since I opened the issue, I need to check if this bug still occurs. For that, I need to open a pull request to someone else's repository.

Is it okay if I one of your repositories, and open a dummy pull request to it?

**Reply by MathiasReker** [reply]
Sure! :)

**Reply by hagai-helman** [reply]
Thanks!

I checked, and they added a "restore branch" button in the pull request page. After one restores the branch, they can reopen the pull request. So the bug is now fixed. Thank you for your help!

**Reply by MathiasReker** [reply]
Great it is solved! Please make this as answered, to close it :-)
